docs/rst/knora-api-server/api_v1/adding-resources.rst

```diff
@@ -132,70 +126,232 @@ The resource's Iri has to be provided in the URL (as its last segment). The new
 The JSON format of the request is described in the TypeScript interface ``changeResourceLabelRequest`` in module ``createResourceFormats``.
 The response is described in the TypeScript interface ``changeResourceLabelResponse`` in module ``createResourceFormats``.
 
-*********************************************
-Adding Multiple Resources in a Single Request
-*********************************************
+Bulk Import
+-----------
 
-Multiple resources can be created in a single request. This is especially
-useful if the resources have links to one another. The entire request will be
-checked for consistency as a whole.
+If you have a large amount of data to import into Knora, it can be more convenient to use
+the bulk import feature than to create resources one by one. In a bulk import operation,
+you submit an XML document to Knora, describing multiple resources to be created.
+This is especially useful if the resources to be created have links to one another.
+Knora checks the entire request for consistency as as a whole, and performs the update
+in a single database transaction.
 
-The resources to be created must be described in an XML file. The XML file
-containing the resource descriptions can be imported directly to Knora by a
-POST request. The request has to be sent to the Knora server using the
-``resources/xml`` path segment.
+The procedure for using this feature is as follows:
 
-::
+1. Make a request to the Knora API server to get XML schemas describing the XML to be provided
+   for the import.
+2. If the project's data includes digital representations to be stored in Sipi, upload those files to
```

Comment:
So we need to upload files to `knora` outside of `knora` so `knora? can find them locally and be able to upload them to `sipi`?